ParseUI/Classes/LogInViewController/PFLogInView.h

```diff
@@ -57,6 +57,20 @@ typedef NS_OPTIONS(NSInteger, PFLogInFields) {
                             PFLogInFieldsDismissButton)
 };
 
+/**
+ `PFLoginFields`'s accessibility label identifiers
+ 
+ @see PFLogInView
+ */
+extern NSString * const PFLogInViewUsernameFieldAccessibilityIdentifier;
```

Comment:
Nit: Per our style - could you remove the space between `*` and `const`, as the star should only be space separated on the left.